src/content/learn/index.md

```diff
@@ -4,7 +4,7 @@ title: Quick Start
 
 <Intro>
 
-Welcome to the React documentation! This page will give you an introduction to the 80% of React concepts that you will use on a daily basis.
+রিএক্ট ডকুমেন্টেশনে স্বাগতম! এই পৃষ্ঠাটি আপনাকে ৮০% ভূমিকা দেবে রিএক্ট এর, যা আপনি প্রতিদিন ব্যবহার করবেন।
```

Comment:
Lets discuss if we should keep the word "React" as it is, or should we call it রিএক্ট or not 

 @moshfiqrony, @ahmmedabir9, @nazir021, @nomandhoni-cs, @shakib609, @nutboltu, @nafistiham

Reply:
Let's keep it "React"

Reply:
I concur with Rony bhai. This is like a brand name which is better left as it is.

Reply:
I also prefer to keep is react, and not only this one. all the key words we can keep same. for example, Components, Props, State, etc.

Reply:
@anwarhossain1 wanna update it to say React, please and thank you?

On a separate note,
We should make a new updated version of this and link it in our guidelines for contributing?
This way everyone will already know what words to avoid writing it phonetically in bengali? 
https://github.com/stale-reactjs-org-translations/bn.reactjs.org/wiki/Glossary

@ahmmedabir9 looking at the glossary above, people chose to write কম্পোনেন্ট for Component.
State and props were left as it is. Same as React!

Reply:
@ahnafyy updated it with React.